packages/editor-ui/src/event-bus/node-view.ts

```diff
@@ -1,3 +1,30 @@
+import type { EventBus } from 'n8n-design-system/utils';
 import { createEventBus } from 'n8n-design-system/utils';
+import type { IDataObject } from 'n8n-workflow';
 
-export const nodeViewEventBus = createEventBus();
+export type CallbackFn = () => void;
+
+export interface NodeViewEventBusEvents {
+	/** Command to create a new workflow */
+	newWorkflow: never;
+
+	/** Command to open the chat */
+	openChat: never;
+
+	/** Command to save the current workflow */
+	saveWorkflow: CallbackFn;
```

Comment:
using this particula alias feels confusing. We should either call the type `OnSaveWorkflowFn` or replace the line with `saveWorkflow: () => void;`

Reply:
Agreed. I tried to clarify it